src/Controls/src/Core/Handlers/Items/Android/Adapters/EmptyViewAdapter.cs

```diff
@@ -273,12 +273,12 @@ bool IsEmpty(int position)
 		int GetHeight(ViewGroup parent)
 		{
 			var headerFooterHeight = parent.Context.ToPixels(_headerHeight + _footerHeight);
-			return Math.Abs((int)(parent.Height - headerFooterHeight));
+			return Math.Abs((int)(parent.MeasuredHeight - headerFooterHeight));
```

Comment:
I don't think `Height` is ever right in these cases

Reply:
yeah i did the same change on iOS